cpanfile

```diff
@@ -53,4 +59,5 @@ on 'develop' => sub {
     requires 'Test::Kwalitee'      => '1.22';
     requires 'Test::Pod::Spelling::CommonMistakes' => '1.000';
     requires 'Try::Tiny'  => '0.24';
+    requires 'Authen::NTLM' => '1.02';
```

Comment:
`dzil` generates this file automatically, so it will clobber any changes when it is next run.  You could have a look at https://metacpan.org/pod/distribution/Dist-Zilla/lib/Dist/Zilla/Plugin/Prereqs.pm to see how to handle this via `dist.ini`